scio-tensorflow/src/main/protobuf/schema.proto

```diff
@@ -1,3 +1,5 @@
+// TODO(brianm): Delete when deprecated schema logic goes away
```

Comment:
@brianmartin instead of having all of this TODO spread across the codebase I think it's better to have an issue that points to what needs to be deprecated. It's easier to track and not forget.

Reply:
👍 #1557